internal/resources/resource_loadbalancer.go

```diff
@@ -37,32 +37,47 @@ func LoadBalancer() *schema.Resource {
 			"enabled": {
 				Type:        schema.TypeBool,
 				Optional:    true,
-				Description: "Network Loadbalancer configuration enabled",
+				Description: "Pass `true` to allow for enabled and Pass `false` to disabled",
 				Default:     true,
 			},
-			"visibility": {
-				Type:        schema.TypeString,
-				Optional:    true,
-				Description: "Network Loadbalancer is public/private visibility mode",
-			},
-			"resource_permission": {
-				Type:        schema.TypeList,
-				Required:    true,
-				Description: "permission access for Loadbalancer",
+			"group_access": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
 				Elem: &schema.Resource{
 					Schema: map[string]*schema.Schema{
 						"all": {
 							Type:        schema.TypeBool,
+							Optional:    true,
 							Default:     true,
+							Description: "Pass `true` to allow access to all groups.",
+						},
+						"sites": {
+							Type:        schema.TypeList,
 							Optional:    true,
-							Description: "If `true` then resource_permission rule will be active/enabled.",
+							Description: "List of sites/groups",
+							Elem: &schema.Resource{
```

Comment:
please validate setting all to false and assigning to only one group.

Reply:
done